SConstruct

```diff
@@ -223,6 +223,7 @@ opts.Add(BoolVariable("vulkan", "Enable the vulkan rendering driver", True))
 opts.Add(BoolVariable("opengl3", "Enable the OpenGL/GLES3 rendering driver", True))
 opts.Add(BoolVariable("d3d12", "Enable the Direct3D 12 rendering driver on supported platforms", False))
 opts.Add(BoolVariable("metal", "Enable the Metal rendering driver on supported platforms (Apple arm64 only)", False))
+opts.Add(BoolVariable("angle", "Enable the GLES3/ANGLE rendering driver on supported platforms", False))
```

Comment:
Is this expected to be `False` by default? Before this PR, ANGLE support is enabled by default (with dynamic library loading).